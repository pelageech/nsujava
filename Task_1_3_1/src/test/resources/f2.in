War and Peace.txt
трус