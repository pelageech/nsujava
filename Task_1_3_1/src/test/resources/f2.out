9
2967 1 37
3258 1 158
3323 2 25 38
4058 1 687
12310 1 67
12655 1 1033
12772 1 507
13749 1 1459
16153 2 48 216